tests/utils.py

```diff
@@ -1,15 +1,16 @@
 from typing import Optional
 
 from fastapi import Response
-from httpx import AsyncClient
+from httpx import AsyncClient, Cookies
 
 from app import models, repository
-from app.main import get_app
-from app.oauth2 import create_access_token
+from app.auth_manager import get_strategy
+from app.config import settings
+from app.main import app
 from app.utils.enums import RequestMethod
 
 
-def authorized_httpx_client(client: AsyncClient, user: models.User):
+async def authorized_httpx_client(client: AsyncClient, user: models.User):
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **issue (testing):** Ensure consistency in async usage across test utilities.

Given that `authorized_httpx_client` is now an async function, ensure all its usages across tests are awaited properly to avoid runtime errors.